layouts/shortcodes/embed-pdf.html

```diff
@@ -128,7 +128,7 @@
 
     // The workerSrc property shall be specified.
     if (pdfjsLib.GlobalWorkerOptions.workerSrc == '')
-      pdfjsLib.GlobalWorkerOptions.workerSrc = "{{.Site.BaseURL}}" + 'js/pdf-js/build/pdf.worker.js';
+      pdfjsLib.GlobalWorkerOptions.workerSrc = "{{.Site.BaseURL}}" + '/js/pdf-js/build/pdf.worker.js';
```

Comment:
Were you able to test this locally with this change?
I think the leading slash wasn't added as the BaseURL usually has a trailing slash. 
If it did work for you please let me know.

Reply:
Yes, this solution works with or without a / the baseURL, both locally and deployed. 
// must be normalised to just / somewhere down the line.